src/test/java/com/commercetools/sync/sdk2/products/utils/ProductSyncMockUtils.java

```diff
@@ -202,15 +237,76 @@ public static CategoryOrderHints createRandomCategoryOrderHints(
+    List<PriceDraft> priceDrafts = createPriceDraft(productVariant.getPrices());
+    return ProductVariantDraftBuilder.of()
+                                     .assets(assetDrafts)
+                                     .attributes(productVariant.getAttributes())
+                                     .images(productVariant.getImages())
+                                     .prices(priceDrafts)
+                                     .sku(productVariant.getSku())
+                                     .key(productVariant.getKey())
+        .build();
+  }
+
+  public static List<AssetDraft> createAssetDraft(List<Asset> assets) {
+    return assets.stream()
+        .map(asset -> AssetDraftBuilder.of()
+                                       .custom(createCustomFieldsDraft(asset.getCustom()))
+                                       .description(asset.getDescription())
+                                       .name(asset.getName())
+                                       .key(asset.getKey())
+                                       .sources(asset.getSources())
+                                       .tags(asset.getTags()).build())
+        .collect(Collectors.toList());
+  }
+
+  public static List<PriceDraft> createPriceDraft(List<Price> prices) {
+    return prices.stream().map(price -> PriceDraftBuilder.of()
+    .channel(ChannelResourceIdentifierBuilder.of().id(price.getChannel().getId()).build())
+    .country(price.getCountry())
+    .custom(createCustomFieldsDraft(price.getCustom()))
+    .customerGroup(CustomerGroupResourceIdentifierBuilder.of().id(price.getId()).build())
+    .discounted(createDiscountedPriceDraft(price.getDiscounted()))
+    .key(price.getKey())
+    .validFrom(price.getValidFrom())
+    .validUntil(price.getValidUntil())
+    .tiers(createPriceTierDraft(price.getTiers()))
+    .value(price.getValue()).build()).collect(Collectors.toList());
+  }
+
+  public static DiscountedPriceDraft createDiscountedPriceDraft(DiscountedPrice discountedPrice) {
+    return DiscountedPriceDraftBuilder.of()
+    .discount(discountedPrice.getDiscount())
+    .value(discountedPrice.getValue())
+    .build();
+  }
+
+  public static List<PriceTierDraft> createPriceTierDraft(List<PriceTier> priceTiers) {
+    return priceTiers.stream().map(priceTier -> PriceTierDraftBuilder.of()
+                                                                     .minimumQuantity(priceTier.getMinimumQuantity())
+                                                                     .value(priceTier.getValue())
+                                                                     .build())
+        .collect(Collectors.toList());
+  }
+
+  public static CustomFieldsDraft createCustomFieldsDraft(CustomFields customFields) {
+    return CustomFieldsDraftBuilder.of()
+                            .fields(customFields.getFields())
+                            .type(customFields.getType().toResourceIdentifier())
+                            .build();
   }
```

Comment:
I would put this method into a new class `AssetUtils` which has separate role from this class

Reply:
[837a7a9](https://github.com/commercetools/commercetools-sync-java/pull/886/commits/837a7a9c3750b41c90c179a32dd6e69e1e7ad68a)